src/test/fuzz/base_encode_decode.cpp

```diff
@@ -67,9 +66,9 @@ FUZZ_TARGET(base32_encode_decode)
         assert(encoded_string == ToLower(TrimStringView(random_string)));
     }
     // Encode/Decode roundtrip
-    const auto encoded{EncodeBase32(buffer)};
+    const auto encoded{EncodeBase32(random_string)};
```

Comment:
this is changed back and fourth between the two commits, probably a rebase error.

Reply:
Thanks for the review. It was a rebase error indeed, thanks for catching it.